scripts/bd_to_prod.sh

```diff
@@ -41,15 +41,14 @@ echo "Getting latest changes from git"
 changes=$(git log $(git describe --tags --abbrev=0)..HEAD --oneline)
 
 
-
-sudo pip install --upgrade bumpversion
-bumpversion patch --message "{current_version} → {new_version}. Changes: ${changes}"
-
-
 echo "Uploading to PyPi"
```

Comment:
upload the "previous" version to pypi, and not the one that we're writing now (https://lumigo.atlassian.net/browse/RD-3309)